cmd/release-controller/main.go

```diff
@@ -178,6 +180,8 @@ func main() {
 
 	flagset.StringVar(&opt.Registry, "registry", opt.Registry, "Specify the registry, that the artifact server will use, to retrieve release images when located on remote clusters")
 
+	flagset.StringArrayVar(&opt.ClusterGroups, "cluster-group", opt.ClusterGroups, "A comma seperated list of build cluster names to evenly distribute jobs to.  May be specified multiple times to account for different configurations of build clusters.")
```

Comment:
Do we have a use-case for not having all build clusters equal?  This would be simpler later on if we only had one list.

Reply:
> Do we have a use-case for not having all build clusters equal? This would be simpler later on if we only had one list.

You explained this as restrictions on a few jobs.  Please add a code comment explaining why we need this power.  I believe you specifically mentioned vpshere.

Reply:
Done!